src/vs/workbench/contrib/scm/browser/scmViewPane.ts

```diff
@@ -316,19 +316,19 @@ class InputRenderer implements ICompressibleTreeRenderer<ISCMInput, FuzzyScore,
 			}
 		};
 
-		const initialRender = () => {
+		const startListeningContentHeightChange = () => {
 			disposables.add(templateData.inputWidget.onDidChangeContentHeight(onDidChangeContentHeight));
-			onDidChangeContentHeight();
-		};
 
-		const contentHeight = templateData.inputWidget.getContentHeight();
+			const contentHeight = templateData.inputWidget.getContentHeight();
 
-		if (contentHeight !== InputRenderer.DEFAULT_HEIGHT) {
-			const timeout = setTimeout(initialRender, 0);
-			disposables.add({ dispose: () => clearTimeout(timeout) });
-		} else {
-			initialRender();
-		}
+			if (contentHeight !== InputRenderer.DEFAULT_HEIGHT) {
+				onDidChangeContentHeight();
+			}
+		};
+
+		// Setup height change listener on next tick
+		const timeout = setTimeout(startListeningContentHeightChange, 0);
+		disposables.add({ dispose: () => clearTimeout(timeout) });
```

Comment:
We have a `toDisposable` helper for this

Reply:
This somehow survived the huge refactoring. We actually have a `disposableTimeout` helper which is even better. Will change it in `master`!

Reply:
https://github.com/microsoft/vscode/commit/5278689345deb4758421f3da3d87ce21d7e43697